README.md

```diff
@@ -1 +1,55 @@
-Demo to practice making packages with poetry, git hooks and FastAPI
\ No newline at end of file
+# Demo FastAPI
+A demo repository to practice making packages with poetry, git hooks and FastAPI. \
+Following the [API tutorial](https://github.com/Naeemedmo/python-playground) by [Naeemedmo](https://github.com/Naeemedmo).
+
+## About
+We made an API called **sales** that takes orders and saves them in a relational database. \
+More specifically, API takes orders of apples and oranges, assigns a unique id to each order and saves it to the SQL database at a MS SQL docker container.
+
+Orders contain:
+
+- **When**: Required - e.g. 2023/04/19 (Given date is in string format YYYY/MM/DD happened after 2000/01/01)
+- **Who**: Required - e.g. Ana (Name cannot contain numbers)
+- **What**: Optional (at least one of them) - e.g. 2 Apples 3 Oranges Or 2 Apples Or 3 Oranges
+
+Technicalities:
+- API is created using [FastAPI](https://fastapi.tiangolo.com/) and [pydantic](https://docs.pydantic.dev/latest/)
+- The database is saved on a MS SQL docker container using [latest docker image](https://hub.docker.com/_/microsoft-mssql-server)
```

Comment:
Terminology tip: The image is a "Microsoft SQL Server". A server can host one or many databases.